kinetic/distribution.yaml

```diff
@@ -5028,6 +5028,23 @@ repositories:
       url: https://github.com/astuff/ibeo_lux.git
       version: release
     status: developed
+  ifm3d-release:
+    doc:
+      type: git
+      url: https://github.com/ifm/ifm3d.git
+      version: master
+    release:
+      packages:
+      - ifm3d-core
```

Comment:
Also, I think every other package in all of the distribution.yaml files uses `ifm3d_core`, so I'd suggest changing your package name to that.

Reply:
Yup you're right -- REP144 mandates I should use an _ instead of -. I'll correct.